trillian/ctfe/requestlog.go

```diff
@@ -73,56 +73,57 @@ type DefaultRequestLog struct {
 }
 
 // Start logs the start of request processing.
-func (dlr *DefaultRequestLog) Start(ctx context.Context) {
+func (dlr *DefaultRequestLog) Start(ctx context.Context) context.Context {
 	glog.V(vLevel).Info("RL: Start")
+	return ctx
 }
 
 // LogPrefix logs the prefix of the CT log that this request is for.
-func (dlr *DefaultRequestLog) LogPrefix(p string) {
+func (dlr *DefaultRequestLog) LogPrefix(c context.Context, p string) {
```

Comment:
Context parameters are normally named `ctx` -- any reason not to stick to that here (and throughout)?

Reply:
(or alternatively use `_` as this incarnation of the interface doesn't use the param)

Reply:
I've gone with '_' as this impl doesn't keep state in the context.